sirbot/slack/database/sqlite/user.py

```diff
@@ -7,18 +7,25 @@
 async def add(db, user):
     await db.execute(
         '''INSERT OR REPLACE INTO slack_users
-         (id, dm_id, admin, raw, last_update)
-         VALUES (?, ?, ?, ?, ?)''',
+         (id, dm_id, admin, raw, last_update, deleted)
+         VALUES (?, ?, ?, ?, ?, ?)''',
         (user.id, user.dm_id, user.admin, json.dumps(user.raw),
-         user.last_update))
+         user.last_update, user.deleted))
+
+
+async def add_multiple(db, users):
+    for user in users:
+        await add(db, user)
 
 
 async def delete(db, id_):
     await db.execute('''DELETE FROM slack_users WHERE id = ? ''', (id_,))
 
 
 async def get_all(db):
-    pass
+    await db.execute('''SELECT slack_users''')
+    users = await db.fetchall()
+    return users
```

Comment:
Good start :)

Remember we want to be able to pass the `deleted` parameter to `get_all` so the work is done by the db.

Here are some links that might help you:

- https://www.tutorialspoint.com/sqlite/sqlite_select_query.htm
- https://www.tutorialspoint.com/sqlite/sqlite_where_clause.htm